### DISCUSSION TITLE
custom nextjs server tailwind does not work

### DISCUSSION THREAD

**Discussion by ivanjeremic** [vercel/next.js#66598] [Help]
### Summary

If I do my custom nextjs server tailwind does not work, anyone idea why? 
```js
import next from "next";

const dev = process.env.NODE_ENV !== "production";
const app = next({
  dev,
  dir: "../ui",
});
const handle = app.getRequestHandler();

```

**Comment by Edwin-Luijten** [comment]
I have the same issue, this is my server implementation:

```javascript
const cnf = await loadConfig(PHASE_DEVELOPMENT_SERVER, FRAMEWORK_PATH);

const nx = next({
        conf: cnf,
        customServer: true,
        dev: true,
        dir: FRAMEWORK_PATH
    }
);
await nx.prepare();

const app = express();
const server = createServer(app);

app.use('/', express.static(path.join(FRAMEWORK_PATH, 'public')));
app.all('*', (req, res) => nx.getRequestHandler()(req, res));

server.listen(3000, () => {
    console.log('💥 Doc Forge is running at http://localhost:3000');
    console.log('Press Ctrl+C to stop');

    const onExit = () => process.exit(0);

    process.on('SIGINT', onExit);
    process.on('SIGTERM', onExit);
});
```

When running it, it starts to build and then pops-up this message:
> warn - The `content` option in your Tailwind CSS configuration is missing or empty.
> warn - Configure your content sources or your generated CSS will be missing styles.
> warn - https://tailwindcss.com/docs/content-configuration

The root of the project contains a tailwind.config.mjs and postcss config.

When I run the native next dev command it all works correctly, so there must be something missing when using our own implementation.

Using next 14.2.4

PS.

```javascript
await nextDev({
    port: 3000,
    hostname: 'localhost',
}, 'cli', FRAMEWORK_PATH)
```

Gives the same result, this command is directly used in the nextjs cli as well:
https://github.com/vercel/next.js/blob/canary/packages/next/src/bin/next.ts#L184
